### ISSUE TITLE
Style snooping of controls of loaded application is missing

### ISSUE THREAD

**Issue by talbotsk** [drewnoakes/style-snooper#7]
Hi, when I press "Load" button at the right top corner of the application it loads exe, but it snoops only the application style. Not the styles of used controls. E.g. I wanted to obtain the style of DevExpress dxe:TextEdit control, but with no success. Would it be possible to integrate this functionality? Thank you.

**Comment by drewnoakes** [comment]
Is it possible that DevExpress styles are stored in separate assemblies, perhaps for theming purposes?

**Comment by talbotsk** [comment]
Yes, they are in separate assemblies. I tryied to access them in 
`        private static IEnumerable<Type> GetFrameworkElementTypesFromAssembly(Assembly assembly)`
via
`        assembly.GetReferencedAssemblies()`
but this method didn't returned all referenced assemblies. I do not know why.
Now I realized I can load dll too. So I extracted styles directly from
`   DevExpress.Xpf.Core.v18.1.dll`
assembly. Thank you for your great app! :-)
Problem solved.
